mne/time_frequency/tfr.py

```diff
@@ -798,6 +801,10 @@ def tfr_multitaper(inst, freqs, n_cycles, time_bandwidth=4.0,
                    n_jobs=1, picks=None, average=True, verbose=None):
     """Compute Time-Frequency Representation (TFR) using DPSS tapers.
 
+    Same computation as the function tfr_array_multitaper.
+    Difference: tfr_multitaper takes epochs in the mne.Epochs format as input;
+    tfr_array_multitaper takes epochs in the numpy.array format as input.
```

Comment:
```suggestion
    Same computation as `~mne.time_frequency.tfr_array_multitaper`, but
    operates on `~mne.Epochs` objects instead of
    :class:`NumPy arrays <numpy.ndarray>`.
```